src/apps/sources/pages/ViewSourcePage.tsx

```diff
@@ -154,7 +216,10 @@ export const mapStateToProps = (state: AppState) => ({
 export const mapDispatchToProps = {
   retrieveSourceAndDetails: retrieveSourceAndDetailsAction,
   retrieveConceptsSummary: retrieveConceptsAction,
-  retrieveActiveConceptsSummary: retrieveActiveConceptsAction
+  retrieveActiveConceptsSummary: retrieveActiveConceptsAction,
+  createSourceVersion: createSourceVersionAction,
```

Comment:
@rakesh-1010 Can we write test cases for the newly added props ??

Reply:
Sure bindu added